modules/logging-release-notes-5-6-17.adoc

```diff
@@ -0,0 +1,50 @@
+// module included in /logging/logging-5-6-release-notes
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-6-17_{context}"]
+= Logging 5.6.17
+This release includes link:https://access.redhat.com/errata/RHSA-2024:1507[Logging Bug Fix 5.6.17]
```

Comment:
The advisory this links to doesn't mention 5.6.17, but I'm assuming that it's fine and the correct one anyways.

Reply:
Just rechecked to make sure it's correct! :) It links to this errata: https://errata.devel.redhat.com/advisory/127993